### PR TITLE
Add Afrikaans and Xhosa Translation

### PR DESCRIPTION
**Pull request by iCE3X-CryptoNinja**
Afrikaans and Xhosa are two significant languages for South African Region. This pull request adds a translation for both

### CODE REVIEW

**Comment by iCE3X-CryptoNinja** [comment]
Is there anything we could do to help these pass the checks and be included?

File: loc/ZAR_Afr.js [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+      error: 'U invoer het misluk. Maak asseblief seker u data is korrek en geldig.',
+      success: 'Suksesvol',
+      do_import: 'Invoer',
+      scan_qr: 'of skandeer QR kode?',
+    },
+    scanQrWif: {
+      go_back: 'Gaan Terug',
+      cancel: 'Kanseleer',
+      decoding: 'Decoding',
+      input_password: 'Input password',
```
Comment (CjS77) [inline loc/ZAR_Afr.js]:
```suggestion
      input_password: 'Invoer wagwoord',
```

File: loc/ZAR_Afr.js [hunk 2]
```diff
@@ -0,0 +1,236 @@
+﻿module.exports = {
+  _: {
+    storage_is_encrypted: 'U geheue spasie is nou ge-enkripteer. ‘n Wagwoord word benodig om toegang te verkry. ',
+    enter_password: 'Sleutel wagwoord in',
```
Comment (CjS77) [inline loc/ZAR_Afr.js]:
```suggestion
    enter_password: 'Invoer wagwoord',
```

File: loc/ZAR_Afr.js [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+      copiedToClipboard: 'Gekopieer na klipbord.',
+      label: 'Beskrywing',
+      create: 'Skep',
+      setAmount: 'Bedrag ontvang',
+    },
+  },
+  buyBitcoin: {
+    header: 'Koop Bitcoin',
+    tap_your_address: 'Raak aan die adres om dit na die klipbord the stuur:',
+    copied: 'Gekopieer na klipbord!',
```
Comment (CjS77) [inline loc/ZAR_Afr.js]:
```suggestion
    copied: 'Gestuur na klipbord!',
```

File: loc/ZAR_Afr.js [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+  },
+  settings: {
+    header: 'instellings',
+    plausible_deniability: 'Geloofwaardige ontkenbaarheid...',
+    storage_not_encrypted: 'Berging: Nie-geenkripteer nie',
+    storage_encrypted: 'Berging: Ge-enkripteer',
+    password: 'Wagwoord',
+    password_explain: 'Skep die wagwoord wat u sal gebruik om u berging te de-enkripteer',
+    retype_password: 'Hervoer wagwoord',
+    passwords_do_not_match: 'Wagwoorde stem nie oor een nie',
```
Comment (CjS77) [inline loc/ZAR_Afr.js]:
```suggestion
    passwords_do_not_match: 'Wagwoorde stem nie ooreen nie',
```

File: loc/ZAR_Afr.js [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+    encrypt_storage: 'Enkripteer Berging',
+    lightning_settings: 'Lightning Instellings',
```
Comment (CjS77) [inline loc/ZAR_Afr.js]:
```suggestion
    lightning_settings: 'Blitsbeurs Instellings',
```

File: loc/ZAR_Afr.js [hunk 6]
```diff
@@ -0,0 +1,236 @@
...
+    lightning_settings_explain:
+      'Om u eie LND node te konnekteer, installeer asseblief LndHub' +
+      ' and put its URL here in settings. Leave blank om die standaard LndHub' +
```
Comment (CjS77) [inline loc/ZAR_Afr.js]:
```suggestion
      ' en sit die URL hier in die voorkeure. Los uit om die standaard LndHub' +
```

File: loc/ZAR_Afr.js [hunk 7]
```diff
@@ -0,0 +1,236 @@
...
+    choose_source_wallet: 'Kies ‘n bron beursie',
+    refill_lnd_balance: 'Herlaai Lightning beursie',
+    refill: 'Herlaai',
+    withdraw: 'Ontrek',
+    expired: 'Verval',
+    placeholder: 'Faktuur',
+    sameWalletAsInvoiceError: 'U kan nie ‘n faktuur betaal met die selfde beursie waarmee die faktuur geksep is nie.',
+  },
+  pleasebackup: {
+    title: 'Your wallet is created...',
```
Comment (CjS77) [inline loc/ZAR_Afr.js]:
```suggestion
    title: 'Jou beursie is gereed...',
```

File: loc/ZAR_Afr.js [hunk 8]
```diff
@@ -0,0 +1,236 @@
...
+    text:
+      "Please take a moment to write down this mnemonic phrase on a piece of paper. It's your backup you can use to restore the wallet on other device. You can use Electrum wallet on desktop (https://electrum.org/) to restore the same wallet.",
```
Comment (CjS77) [inline loc/ZAR_Afr.js]:
```suggestion
      "Vat asb. 'n oomblik om die mnemonic frase op 'n stuk papier neer te skryf. Díe kan gebruik word om jou beursie te herstel op 'n ander toestel. Jy kan ook 'Electrum Wallet' (https://electrum.org/) gebruik om u beursie op 'n tafelrekenaar te herstel.",
```

File: loc/ZAR_Afr.js [hunk 9]
```diff
@@ -0,0 +1,236 @@
...
+    ok: 'OK, I wrote this down!',
```
Comment (CjS77) [inline loc/ZAR_Afr.js]:
```suggestion
    ok: 'OK, ek het hulle al neergeskryf!',
```

File: loc/ZAR_Afr.js [hunk 10]
```diff
@@ -0,0 +1,236 @@
...
+  },
+  lndViewInvoice: {
+    wasnt_paid_and_expired: 'This invoice was not paid for and has expired',
+    has_been_paid: 'This invoice has been paid for',
+    please_pay: 'Please pay',
+    sats: 'sats',
+    for: 'For:',
```
Comment (CjS77) [inline loc/ZAR_Afr.js]:
```suggestion
    for: 'Vir:',
```

File: loc/ZAR_Afr.js [hunk 11]
```diff
@@ -0,0 +1,236 @@
...
+    additional_info: 'Additional Information',
```
Comment (CjS77) [inline loc/ZAR_Afr.js]:
```suggestion
    additional_info: 'Addisionele inligting',
```

File: loc/ZAR_Afr.js [hunk 12]
```diff
@@ -0,0 +1,236 @@
...
+    open_direct_channel: 'Open direct channel with this node:',
```
Comment (CjS77) [inline loc/ZAR_Afr.js]:
```suggestion
    open_direct_channel: 'Stel 'n direk konneksie met hierdie nodus op:',
```

File: loc/ZAR_Afr.js [hunk 13]
```diff
@@ -0,0 +1,236 @@
...
+  pleasebackup: {
+    title: 'Your wallet is created...',
+    text:
+      "Please take a moment to write down this mnemonic phrase on a piece of paper. It's your backup you can use to restore the wallet on other device. You can use Electrum wallet on desktop (https://electrum.org/) to restore the same wallet.",
+    ok: 'OK, I wrote this down!',
+  },
+  lndViewInvoice: {
+    wasnt_paid_and_expired: 'This invoice was not paid for and has expired',
+    has_been_paid: 'This invoice has been paid for',
+    please_pay: 'Please pay',
```
Comment (CjS77) [inline loc/ZAR_Afr.js]:
```suggestion
    please_pay: 'Betaal asseblief',
```

File: loc/ZAR_Afr.js [hunk 14]
```diff
@@ -0,0 +1,236 @@
...
+    sameWalletAsInvoiceError: 'U kan nie ‘n faktuur betaal met die selfde beursie waarmee die faktuur geksep is nie.',
+  },
+  pleasebackup: {
+    title: 'Your wallet is created...',
+    text:
+      "Please take a moment to write down this mnemonic phrase on a piece of paper. It's your backup you can use to restore the wallet on other device. You can use Electrum wallet on desktop (https://electrum.org/) to restore the same wallet.",
+    ok: 'OK, I wrote this down!',
+  },
+  lndViewInvoice: {
+    wasnt_paid_and_expired: 'This invoice was not paid for and has expired',
```
Comment (CjS77) [inline loc/ZAR_Afr.js]:
```suggestion
    wasnt_paid_and_expired: 'Hierdie faktuur het verval',
```

File: loc/ZAR_Afr.js [hunk 15]
```diff
@@ -0,0 +1,236 @@
...
+    has_been_paid: 'This invoice has been paid for',
```
Comment (CjS77) [inline loc/ZAR_Afr.js]:
```suggestion
    has_been_paid: 'Hierdie faktuur is opbetaal',
```

**Comment by marcosrdz** [comment]
loc/index.js: 
zar_xho: require('./en.js'),
zar_afr: require('./en.js')


^ Shouldn't those point to their respective language files?

**Comment by fluffypony** [comment]
Aweh my bru

**Comment by ncoelho** [comment]
> Aweh my bru

Mr. Fluffypony :)

Can I steal you for a mail thread instead? Ping me here n@bluewallet.io please

**Comment by limpbrains** [comment]
@CjS77 @iCE3X-CryptoNinja 
Hi! 
I would like to invite you to improve the translation of BlueWallet. There are approximately 200 new phrases. Translation has now moved to [Transifex](https://www.transifex.com/bluewallet/bluewallet).
To participate please register there, find the IC project and join. After confirming the request, you will be able to edit the translation

Thank you!

**Pull request merged**